doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -127,7 +130,7 @@ Additional scenario XML elements:
 ``clos``:
   Class of Service for Cache Allocation Technology. Refer to the SDM 17.19.2 for details and use with caution.
 
-``start_hpa``:
+``start_hpa`` under parent of ``memory``:
```

Comment:
Is this a common formulation: "under parent of"? It sounds odd to me but perhaps it's standard practice to formulate this this way.

Reply:
here "start_hpa" is child element, "memory" is its parent element.

Reply:
Thanks @jsun26intel , it's the way it is written that I found odd, would it still be accurate to say something like "`start_hpa` (a child node of `memory`)"? If so, would it sound better (English-wise)?